shopgate/pwa
# User is shown a white page when navigating

# Description

Updated @virtuous/conductor to v1.0.0-beta.40

## Type of change

Please add an "x" into the option that is relevant:

- [ x ] Bug Fix :bug: (non-breaking change which fixes an issue)
- [  ] New Feature :rocket: (non-breaking change which adds functionality)
- [  ] Breaking Change :boom: (fix or feature that would cause existing functionality to not work as expected)
- [  ] Polish :nail_care: (Just some cleanups)
- [  ] Docs :memo: (Changes in the documentations)
- [  ] Internal :house: Only relates to internal processes.

## How to test it

Select filters inside a category. When pressing 'DONE' then immediately navigate backwards. Then check to see if the search filter route displays correctly.
